stagemonitor/src/main/java/com/cerner/beadledom/stagemonitor/request/LogJsonRequestTraceReporter.java

```diff
@@ -35,16 +31,20 @@
    */
   public LogJsonRequestTraceReporter(ObjectMapper objectMapper) {
     this.logger = LoggerFactory.getLogger(LogJsonRequestTraceReporter.class);
+    this.errorLogger =
+        LoggerFactory.getLogger(LogJsonRequestTraceReporter.class.getSimpleName() + ".Error");
     this.objectMapper = objectMapper;
     this.requestMonitorPlugin = Stagemonitor.getPlugin(RequestMonitorPlugin.class);
     this.jsonRequestTraceLoggerPlugin = Stagemonitor.getPlugin(JsonRequestTraceLoggerPlugin.class);
   }
 
   //@VisibleForTesting
   LogJsonRequestTraceReporter(
```

Comment:
Am I right in saying, that this second constructor is only used during testing?

We don't need to worry about injecting this second logger in from anywhere do we?